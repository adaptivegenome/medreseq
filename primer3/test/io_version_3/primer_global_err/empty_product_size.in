PRIMER_SEQUENCE_ID=empty_product_size
SEQUENCE=TAGCTGATGCTAGCTAGCGTGGGCATGCTAGCTAGCTGATGCAGCTAGCTGATCGTAGCTAGCTATCGATCGTAGCTGACTGATCGTAGCTAGCTGATCGATGCTAGCTAGCTAGCTGATCATCTACGTAGCTGACTATCGATGCTACTACTATCGATCGTAGCTAGCTGATCGTAGCTAGCTAGCTGATGCTAC
PRIMER_DEFAULT_PRODUCT=
=
