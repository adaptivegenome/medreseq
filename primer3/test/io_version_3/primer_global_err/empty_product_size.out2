../src/primer3_core: Empty value for PRIMER_PRODUCT_SIZE_RANGE
