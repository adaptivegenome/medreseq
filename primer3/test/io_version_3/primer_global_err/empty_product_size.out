PRIMER_SEQUENCE_ID=empty_product_size
SEQUENCE=TAGCTGATGCTAGCTAGCGTGGGCATGCTAGCTAGCTGATGCAGCTAGCTGATCGTAGCTAGCTATCGATCGTAGCTGACTGATCGTAGCTAGCTGATCGATGCTAGCTAGCTAGCTGATCATCTACGTAGCTGACTATCGATGCTACTACTATCGATCGTAGCTAGCTGATCGTAGCTAGCTAGCTGATGCTAC
PRIMER_DEFAULT_PRODUCT=
PRIMER_ERROR=Empty value for PRIMER_PRODUCT_SIZE_RANGE
=
